assets/src/stories-editor/components/editor-carousel/indicator.js

```diff
@@ -58,11 +58,12 @@ class Indicator extends Component {
 	}
 
 	getInsertIndex( position ) {
-		const { index } = this.props;
+		let { index } = this.props;
 
 		if ( index !== undefined ) {
-			return position.x === 'right' ? index + 1 : index;
+			index = position.x === 'right' ? index + 1 : index;
 		}
+		return index;
```

Comment:
Could you clarify why was this change necessary?

Reply:
The new ESLint rules I added require always having an explicit return.

Although here I'd just `return undefined` instead of `return index`, even though it's technically the same. But it's eaiser to scan.

Reply:
Yep, we could just use return here instead of the other changes above.

Reply:
code updated